### ISSUE TITLE
Product Backlog

### ISSUE THREAD

**Issue by TTcrockeTT** [PeterBrunone/peters-agile-materials#27]
The slide has a too much feel.
All known
BIIIIIIG

The Product Backlog is minimal but sufficient.

What do you think?

**Comment by PeterBrunone** [comment]
Agreed (about this one and likely others as well). I always want to move toward less text in general, so the slide becomes a prompt rather than a substitute for speaker-audience interaction.

**Comment by TTcrockeTT** [comment]
My apologies. I was not being clear. I was taking issue with the concept of a large product backlog or that it captures all of anything. I was suggesting that it be phrased in a way that delivers the message that the backlog is minimal but sufficient.

**Comment by PeterBrunone** [comment]
Ah. I can see clearly now; my brain is gone.

Yeah, I get what you're saying. Nothing like a too-large backlog to keep you from seeing anything.
